src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-advanced-fieldset/form-advanced-fieldset.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cd-form-advanced-fieldset',
```

Comment:
shall we apply this in other forms as well? like the pools, or rbd image? if not in this PR, then we need a tracker to do it, but probably its better to do it here  in a separate commit? wdyt?

Reply:
Yep, the tracker has a subtask!

Reply:
yeah, osd is not needed. Its only needed in the RBD image. I don't see it in the pool form though. It has some way of progressive disclosure.